photon-core/src/main/java/org/photonvision/vision/processes/VisionRunner.java

```diff
@@ -70,6 +74,7 @@ public void startProcess() {
 
     private void update() {
         while (!Thread.interrupted()) {
+            parentModule.processSettingChanges();
```

Comment:
I'd prefer if this function stayed in a helper class like VisionModuleChangeSubscriber was, or otherwise in VisionRunner, instead of in VisionModule. It seems smelly to delegate this back to the parent module.